### ISSUE TITLE
"Control Texture" feature request

### ISSUE THREAD

**Issue by WarnerV** [kmkolasinski/AwesomeBump#147]
(as discussed via mail)

To use the output textures in Game Engines, it's useful to combine various B&W textures into one texture, reducing memory use.

It would be great if in the output section of AwesomeBump there would be a possibility to 'map' the textures to select-able channels. The mapping could look something like this:

For each Output Texture 1, 2 and 3 (you would never need more than 3): 

-  Red:  (dropdown: select Diffuse (D), Normal (N), Heightmap (H), Metalic (M), Roughness (R), Specular (S), Ambient Occlusion (O)) 
 - Green: (dropdown, locked if Diffuse or Normal are selected in Red, otherwise select H, M, R, S, O)
 - Blue: (dropdown, same as green)
 - Alpha: (dropdown,  select H, M, R, S, O)

This would replace the current functionality where you can already map H and S to alphas.

(by the way, sorry it took longer than expected to write down the GitHub Issue..)

**Comment by kmkolasinski** [comment]
Hi and thanks for pinging me.

**Comment by melvixiodf** [comment]
This would have been a awesome feature to add. I am aware this software is no longer been maintained but it is a very awesome software. 
Also I think the UI of Awesome Bump 5.1 would be better with the Icons for the texture types tabs replaced with labels like in Awesome Bump 4 like Diffuse, metallic instead of the icons if development is ever revived. Would make things more straight forward than having to second guess if you are tweaking the right texture especially for new users.

Also the ability to enter digits in the parameters would also be helpful.

@kmkolasinski Thanks for still making this software available to other 3d artists to use. It is a powerful app.

Cheers.

**Comment by kmkolasinski** [comment]
Yes, I no longer have time to work on this project, however I will at least try to make it work on newer systems. If you don't need features added after version 4, you can still download old released in the Releases page: https://github.com/kmkolasinski/AwesomeBump/releases - I believe they should work too.

**Comment by melvixiodf** [comment]
The new features in 5.1 are useful. No worries, I am sure new users will get used to it.
Any chance you might know what is causing this:
![be488990cb2908f3b2548be2de267f2929446e31](https://user-images.githubusercontent.com/54470864/134822589-525c88f5-7800-44b4-bdf6-82bff1e4c94a.png)

It seems to work for some Color id images and doesn't for most.

**Comment by melvixiodf** [comment]
Ok. Thanks. 
Btw, Is there a way to tweak saturation of the diffuse?

**Comment by kmkolasinski** [comment]
I think there is no explicit slider to control the saturation, but you can easily do it on some other software before editing it in AB

**Comment by melvixiodf** [comment]
Ok. Thanks.
